src/common/snippets/docs/debug_capabilities/LIR_passes_serialization.md

```diff
@@ -0,0 +1,20 @@
+# Snippets LIR passes serialization
+
+LIR(Linear Intermediate Representation) is used as graph reprsentation in control flow pipeline, where dozens of passes are applied to LIR. This is to transfer the graph gradually to the stage that can generate kernel directly via expression instruction emission. When each pass is applied to LIR, there are some expected changes. Developers maybe want to check if the result is as expected. This capability is introduced to serialize LIRs before and after passes, then developer can check these LIR changes and status.
+
+To turn on snippets LIR passses serialization feature, the following environment variable should be used:
+```sh
+    OV_SNIPPETS_DUMP_LIR=<space_separated_options> binary ...
+```
+
+Examples:
+```sh
+    OV_SNIPPETS_DUMP_LIR="passes=ExtractLoopInvariants dir=path/dumpdir formats=all" binary ...
+    OV_SNIPPETS_DUMP_LIR="passes=all dir=path/dumpdir formats=control_flow" binary ...
+    OV_SNIPPETS_DUMP_LIR="passes=Insert formats=data_flow" binary ...
+```
+
+Option names are case insensitive, the following options are supported:
+ - `passes` : Dump LIR around the pass if the value is pass name or is included in pass name. Key word 'all' means to dump LIR around every pass. This option is a must have, should not be omitted.
```

Comment:
Could you please make sure that if this option is not set, the exception is thrown? Although I could miss something, I didn't find the code which performs the corresponding check.

Reply:
It is checked right after read and pass environment var here: https://github.com/openvinotoolkit/openvino/pull/24293/files#diff-1186d874b98cf30339ed37e5036846d4f72946335a6472aad73151d733c7b577R23